addons/nametags/stringtable.xml

```diff
@@ -524,5 +524,13 @@
             <Chinese>強制顯示在準心指到和按按鍵時</Chinese>
             <Turkish>Göstermeyi Yalnızca İmleç ve Tuşa Basmaya Zorla</Turkish>
         </Key>
+        <Key ID="STR_ACE_NameTags_AmbientBrightnessAffectViewDist_DisplayName">
+            <English>Ambient brightness affect view distance</English>
```

Comment:
Rest of this component's settings use title case for display names. For consistency, this setting should too.

Reply:
Actually not. But ok.

https://github.com/acemod/ACE3/blob/cd0dddbf2b4c3ca8098612e7150f3f569c36253c/addons/nametags/stringtable.xml#L173-L174

https://github.com/acemod/ACE3/blob/cd0dddbf2b4c3ca8098612e7150f3f569c36253c/addons/nametags/stringtable.xml#L342-L343

Reply:
Ah, this component is mixed too. But yes, title case would be preferable.